zeebe/protocol/revapi.json

```diff
@@ -90,6 +90,30 @@
           "old": "field io.camunda.zeebe.protocol.record.value.AuthorizationResourceType.USER_TASK",
           "code": "java.field.removed"
         },
+        {
+          "ignore": true,
+          "code": "java.method.removed",
+          "old": "method io.camunda.zeebe.protocol.record.value.PermissionAction io.camunda.zeebe.protocol.record.value.AuthorizationRecordValue::getAction()",
+          "justification": "Action is not required. The record was released with 8.6 without any way to write it. Because of this we can safely remove it."
+        },
+        {
+          "ignore": true,
+          "code": "java.method.removed",
+          "old": "method io.camunda.zeebe.protocol.record.value.ImmutableAuthorizationRecordValue.Builder io.camunda.zeebe.protocol.record.value.ImmutableAuthorizationRecordValue.Builder::withAction(io.camunda.zeebe.protocol.record.value.PermissionAction)",
+          "justification": "Action is not required. The record was released with 8.6 without any way to write it. Because of this we can safely remove it."
+        },
+        {
+          "ignore": true,
+          "code": "java.method.removed",
+          "old": "method io.camunda.zeebe.protocol.record.value.PermissionAction io.camunda.zeebe.protocol.record.value.ImmutableAuthorizationRecordValue::getAction()",
+          "justification": "Action is not required. The record was released with 8.6 without any way to write it. Because of this we can safely remove it."
+        },
+        {
+          "ignore": true,
+          "code": "java.method.removed",
+          "old": "method io.camunda.zeebe.protocol.record.value.ImmutableAuthorizationRecordValue io.camunda.zeebe.protocol.record.value.ImmutableAuthorizationRecordValue::withAction(io.camunda.zeebe.protocol.record.value.PermissionAction)",
+          "justification": "Action is not required. The record was released with 8.6 without any way to write it. Because of this we can safely remove it."
+        },
```

Comment:
❓ Should this lines be added into the `ignored-changes.json` file ? According to https://github.com/camunda/camunda/wiki/Breaking-Changes

Reply:
Done